README.md

```diff
@@ -45,7 +45,7 @@ Switch the `global.mode` to `"mist"` in the `main.js`.
 
 For development we start the interface with a Meteor server for autoreload etc. Start the interface in a separate terminal window:
 
-    $ cd interface && meteor
+    $ cd mist/interface && meteor
```

Comment:
```Start the interface in a separate terminal window``` left me in ~/ But I was being nitpicky

Honestly I was expecting to to more edits but I ran out of day.